spec/controllers/new_administrateur/groupe_instructeurs_controller_spec.rb

```diff
@@ -117,6 +117,12 @@
       it { expect(flash.alert).to be_present }
       it { expect(response).to redirect_to(procedure_groupe_instructeur_path(procedure, procedure.defaut_groupe_instructeur)) }
     end
+
+    context 'of an empty string' do
+      let(:new_instructeur_emails) { nil }
```

Comment:
Si tu mets `context 'of an empty string'`, ça ne devrait pas être plutôt `let(:new_instructeur_emails) { '' }` ?